frontend/src/store/NotificationState.ts

```diff
@@ -3,27 +3,78 @@
     It is structured this way so it doesn't stack multiple messages with the same severity (by design).
     It is intended to be used in conjunction with the NotificationStack component.
 */
-import type { Severity } from "@/enum/SeverityEnum";
+
 import { ref } from "vue"
+import { isDesktop, screenSize } from "@/store/screenSizeState";
+import type { Severity } from "@/enum/SeverityEnum";
 
 const defaultNotification = {
     success: "",
     warn: "",
     error: "",
 };
 
+export const newMsgList = ref();
+export const newFormData = ref();
+export const newSeverity = ref();
+
 export const notifications = ref(
     JSON.parse(JSON.stringify(defaultNotification))
 );
 
 export const pushNotification = (severity: Severity, message: string) => {
     notifications.value[severity] = message;
-}
+};
 
 export const clearNotification = (severity: string) => {
     notifications.value[severity] = '';
-}
+};
 
 export const resetNotification = () => {
     notifications.value = JSON.parse(JSON.stringify(defaultNotification));
-}
+};
+
+//how many full id will be shown per message
+const idPerMsg = () => {
+    if(screenSize <= 725) {
+        return 1;
+
+    } else if(screenSize <= 1060 && isDesktop() || screenSize <= 820) {
+        return 2;
+
+    } else if(screenSize <= 1130 && isDesktop() || screenSize <= 880) {
+        return 3;
+
+    } else if(screenSize <= 1290 && isDesktop() || screenSize <= 960) {
+        return 4;
+
+    } else {
+        return 5;
+
+    }
+};
+
+export const setNotificationMsg = (msgList: string[], formData: any, severity: Severity) => {
```

Comment:
we don't use formData parameter at all, and please rename the msgList, I feel it's actually the forest client number list?

Reply:
I'll teste if it works without the formData

Reply:
the formData is used to get the user id but I'll double check

Reply:
user id you mean the forest client number? but the msgList are for the forest client number as well?

Reply:
the formData is the user name and the msgList it the list with the client ids

Reply:
then let's just pass the user name, and name the parameter that we can understand what it's for